src/main/scala/verification/schedulers/DPORwHeuristics.scala

```diff
@@ -1095,11 +1157,19 @@ class DPORwHeuristics(depth_bound: Option[Int] = None) extends Scheduler with La
   def test(events: Seq[ExternalEvent],
            violation_fingerprint: ViolationFingerprint,
            _stats: MinimizationStats) : Option[EventTrace] = {
+    currentSubsequence = events
+    lookingFor = violation_fingerprint
     stats = _stats
     Instrumenter().scheduler = this
-    // TODO(cs): explore schedules.
+    run(events, initialTrace=Some(_initialTrace),
+        initialGraph=Some(_initialDegGraph))
```

Comment:
I think we actually should supply f1 and f2 here. In particular f2 is useful for storing the depgraph for future runs (DPOR might have built a different one from initial). Also f1 gives us an easy way to get dep.